src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -148,7 +148,12 @@ public UserRole getUserRoleClassifications(String userRole) {
     }
 
     @Override
+    @Cacheable("caseTypeDefinitionLatestVersionCache")
```

Comment:
not 100% sure about it, but if you put the Cacheable directly on the original method without adding this level of indirection it should still work. Needs to be tested though

Reply:
It does work, indeed. The new integration test introduced verifies the cache behaviour.

Reply:
in that case can we remove 
```
public CaseTypeDefinitionVersion getLatestVersion(String caseTypeId) {
        return doGetLatestVersion(caseTypeId);
    }
```
and put the annotation directly on `getLatestVersion`?